mne/minimum_norm/inverse.py

```diff
@@ -1339,10 +1340,11 @@ def make_inverse_operator(info, forward, noise_cov, loose='auto', depth=0.8,
     """  # noqa: E501
     is_fixed_ori = is_fixed_orient(forward)
 
-    if fixed and loose not in [None, 'auto']:
-        warn('When invoking make_inverse_operator with fixed=True, the loose '
-             'parameter is ignored.')
-        loose = None
+    if fixed:
+        if loose not in [None, 'auto', 0.]:
```

Comment:
I would have done:

if loose != 'auto':

Reply:
`None` is deprecated (so will emit warning later) and `loose=0.` means "fixed" so I don't see a problem with including it